api-report/runtime-definitions.api.md

```diff
@@ -262,6 +262,11 @@ export interface IGarbageCollectionState {
     };
 }
 
+// @public
+export interface IHaveEntrypoint {
```

Comment:
i don't think we gain a whole lot by adding this interface, and i'd prefer not to add it unless we really need it, so i would just add the property directly to the interfaces above

Reply:
No strong objections to remove it, my only concern is making sure the three interfaces stay in sync, because they feel relatively independent of each other. I'll push the removal, just wanted to double check that you find it ok to have the property defined in 3 different places (`IDataStore`, `IFluidDataStoreRuntime`, `IFluidDataStoreChannel`).